geographic_msgs/GeoPoint geopoint
time stamp
---
geometry_msgs/PointStamped point